resources/config/services/services.yml

```diff
@@ -21,6 +21,11 @@ services:
     bind:
       $showSubmissionCount: '%application.show_submission_count%'
 
+  OpenCFP\Http\Action\Profile\EditAction:
+    public: true
+    bind:
+      $downloadFromPath: 'foo'
```

Comment:
@derrabus 

Can you give me a hint how I properly bind the `$downloadFromPath` from `PathInterface::downloadFromPath()`?